src/backend/api/trace_endpoints.py

```
@@ -0,0 +1,214 @@
+            query += " AND status = ?"
+            params.append(status.upper())
+
+        query += " ORDER BY created_at DESC LIMIT ? OFFSET ?"
+        params.extend([limit, offset])
+
+        rows = conn.execute(query, params).fetchall()
+        conn.close()
+        return {"traces": [dict(r) for r in rows], "limit": limit, "offset": offset}
+    except FileNotFoundError as e:
+        return {"traces": [], "limit": limit, "offset": offset, "note": str(e)}
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)